content/providers/03-community-providers/20-sap-ai.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: SAP AI Core
+description: SAP AI Core Provider for the AI SDK
+---
+
+# SAP AI Core
```

Comment:
can you add a note that this provider is not from SAP itself but developed by a 3rd party